crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -210,7 +210,7 @@ fn parse_entries_numpy(content: &str) -> Vec<QualifiedName> {
     for potential in split {
         if let Some(first_char) = potential.chars().nth(indentation) {
```

Comment:
Could we use `strip_prefix` here with the indentation instead of using `chars`?

Reply:
Ok I reimplemented using `strip_prefix` but I still needed to call `chars` to check if the first char `is_whitepace`

Reply:
Makes sense!